pkg/config/config.go

```diff
@@ -40,21 +41,21 @@ const (
 
 // Configuration options for the search-collector.
 type Config struct {
-	AggregatorConfig *rest.Config // Config object for hub. Used to get TLS credentials.
-
-	AggregatorConfigFile string `env:"HUB_CONFIG"`         // Config file for hub. Will be mounted in a secret.
-	AggregatorURL        string `env:"AGGREGATOR_URL"`     // URL of the Aggregator, includes port but not any path
-	AggregatorHost       string `env:"AGGREGATOR_HOST"`    // Host of the Aggregator
-	AggregatorPort       string `env:"AGGREGATOR_PORT"`    // Port of the Aggregator
-	ClusterName          string `env:"CLUSTER_NAME"`       // The name of this cluster
-	ClusterNamespace     string `env:"CLUSTER_NAMESPACE"`  // The namespace of this cluster
-	DeployedInHub        bool   `env:"DEPLOYED_IN_HUB"`    // Tracks if deployed in the Hub or Managed cluster
-	HeartbeatMS          int    `env:"HEARTBEAT_MS"`       // Interval(ms) to send empty payload to ensure connection
-	KubeConfig           string `env:"KUBECONFIG"`         // Local kubeconfig path
-	MaxBackoffMS         int    `env:"MAX_BACKOFF_MS"`     // Maximum backoff in ms to wait after error
-	RediscoverRateMS     int    `env:"REDISCOVER_RATE_MS"` // Interval(ms) to poll for changes to CRDs
-	ReportRateMS         int    `env:"REPORT_RATE_MS"`     // Interval(ms) to send changes to the aggregator
-	RuntimeMode          string `env:"RUNTIME_MODE"`       // Running mode (development or production)
+	AggregatorConfig     *rest.Config // Config object for hub. Used to get TLS credentials.
```

Comment:
What happened here? it looks like a little reformatting? I checked and it looks like all the letters are the same as before, so good.

Reply:
yes - should be same as before.